packages/server/src/tenants/getTenant/openapi/paths/getTenant.json

```diff
@@ -0,0 +1,73 @@
+{
+  "get": {
+    "tags": [
+      "tenants"
+    ],
+    "summary": "Get a tenant",
+    "description": "Get a tenant in a database",
+    "operationId": "getTenant",
+    "parameters": [
+      {
+        "name": "workspaceSlug",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "databaseName",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "tenantId",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "A Tenant",
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "../../../../openapi/schemas/Tenant.json"
```

Comment:
TIL this does not generate the correct browser code, but I will fix that, so no worries.